modules/microshift-mirroring-prereqs.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deploy-with-mirror-registry.adoc
+
+:_content-type: CONCEPT
+[id="microshift-configuring-mirroring-prereqs_{context}"]
+= Configuring mirroring prerequisites
+
+You must create a container image registry credentials file that allows the mirroring of images from your Internet-connected mirror host to your air-gapped mirror. Follow the instructions in the "Configuring credentials that allow images to be mirrored" link provided in the Additional resources to create a `~/.pull-secret-mirror.json` file that includes the user credentials for accessing the mirror.
```

Comment:
Can you put a sentence here that says specifically it needs to be present on the mirror registry host ?

Reply:
How about "Follow the instructions in the "Configuring credentials that allow images to be mirrored" link provided in the Additional resources to create a `~/.pull-secret-mirror.json` file _on the mirror registry host_ that includes the user credentials for accessing the mirror."